geoportal/c2cgeoportal_geoportal/scaffolds/__init__.py

```diff
@@ -92,9 +92,13 @@ def pre(  # pylint: disable=arguments-renamed
             package_logger = "app"
         vars_["package_logger"] = package_logger
         vars_["geomapfish_version"] = os.environ["VERSION"]
-        vars_["geomapfish_major_version_tag"] = (
-            os.environ["VERSION"] if vars_.get("unsafe_long_version", False) else os.environ["MAJOR_VERSION"]
+        vars_["geomapfish_version_tag"] = "GEOMAPFISH_VERSION"
+        vars_["geomapfish_version_tag_env"] = "${GEOMAPFISH_VERSION}"
```

Comment:
Not sure to understand these two lines. `..._tag` is variable name and `..._tag_env` is a tag.

Reply:
Maybe:
```
vars_["geomapfish_version_arg"] = "GEOMAPFISH_VERSION"
vars_["geomapfish_version_tag"] = "${GEOMAPFISH_VERSION}"
```

Reply:
Thay are used in the Docker files to shoos the version of the build (and config) image as that:
```
ARG {{geomapfish_version_tag}}
FROM camptocamp/geomapfish-tools:{{geomapfish_version_tag_env}} as builder
```
The _env is needed because 
`FROM camptocamp/geomapfish-tools:${{{geomapfish_version_tag}}} as builder`
is not working...

Reply:
Ok, so is the ARG really necessary ?

Reply:
If it is the case, I would like to have more meaningfull variable names.
Maybe _tag for a tag and _arg for an arg name.

Reply:
but it is not trivial to understand that _tag is for an arg name and _tag_env for a tag.

Reply:
Here it seems that changing the ARG in docker-compose will change something, but as I understand, it will change nothing.

Reply:
Sorry, get it now, {{geomapfish_version_tag_env}} will get replaced by ${GEOMAPFISH_VERSION} by pcreate.